.eslintrc.json

```diff
@@ -38,7 +42,10 @@
       }
     ],
     "class-methods-use-this": "error",
-    "comma-dangle": "error",
+    "comma-dangle": [
+      "error",
+      "always-multiline"
```

Comment:
Couldn't we change this to ""only-multiline", and avoid having to add all those unnecessary commas?!